crypto/asn1/x_int64.c

```diff
@@ -28,13 +28,21 @@
 
 static int uint64_new(ASN1_VALUE **pval, const ASN1_ITEM *it)
 {
-    *(uint64_t *)pval = 0;
+    *pval = (ASN1_VALUE *)OPENSSL_zalloc(sizeof(uint64_t));
+    if (*pval == NULL)
+        return 0;
     return 1;
 }
 
 static void uint64_free(ASN1_VALUE **pval, const ASN1_ITEM *it)
 {
-    *(uint64_t *)pval = 0;
+    OPENSSL_free(*pval);
+    *pval = NULL;
+}
+
+static void uint64_clear(ASN1_VALUE **pval, const ASN1_ITEM *it)
+{
+    **(uint64_t **)pval = 0;
```

Comment:
This is perhaps playing a little fast-and-loose with the spec, in that while **pval is created as an object of type uint64_t, *pval was created as an object of type ASN1_VALUE *.  But, if that sort of distinction was going to cause runtime problems, ~all of the ASN.1 code would be affected, and this change is still a vast improvement over what it's replacing, so I wouldn't worry about it.
(I'll just comment on the one instance, even though there are others.)